src/css/vote-CSS.css

```diff
@@ -0,0 +1,109 @@
+html {
+  display: table;
+}
+
+body {
```

Comment:
If you move these styles to a class, rather than on the `body`. If you have another page, using the same CSS file, these styles will be applied and you might not want that on every page.

The class could be a utility class also..

`u-blue-gradient`?

Then have a separate class for the padding